scripts/run_benchmark.sh

```diff
@@ -10,11 +8,11 @@ if [  -z "${output}" ]; then
     output=$(echo "${output}" | sed 's/pallet\//pallets\//')
 fi
 
-echo "Benchmark: ${pallet}"
-cargo +nightly run --release --features runtime-benchmarks -- benchmark \
+echo "Benchmarking ${pallet}..."
+cargo run --release --features runtime-benchmarks -- benchmark \
   --chain="${chain}" \
-  --steps=100 \
-  --repeat=200 \
+  --steps=50 \
+  --repeat=20 \
```

Comment:
Why this? Tbh I have not yet put any thought into, how these parameters affect the statistics behind the benchmarking.

Reply:
I think too many steps and reps seems to be saturating the weights and give much lower ones.